.travis.yml

```diff
@@ -43,4 +64,12 @@ before_script:
     bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
   fi
 script:
-- composer run-tests
+    - |
+        if [[ "$WP_TRAVISCI" == "phpcs" ]]; then
+          composer phpcs
+        elif [[ "$WP_TRAVISCI" == "phpstan" ]]; then
+          composer require --dev szepeviktor/phpstan-wordpress
+          vendor/bin/phpstan analyze
+        else
+          composer run-tests
+        fi
```

Comment:
You'll run into failures with the addition of Mozart, you can have a look at what I did there to keep Travis running as expected: https://github.com/wp-media/wp-rocket/pull/3008/files#diff-354f30a63fb0907d4ad57269548329e3